docs/how-to-deploy-scalardb-cluster.md

```diff
@@ -0,0 +1,67 @@
+# How to deploy ScalarDB Cluster
+
+This document explains how to deploy ScalarDB Cluster using Scalar Helm Charts. For more details on the custom values file for ScalarDB Cluster, see [Configure a custom values file for ScalarDB Cluster](./configure-custom-values-scalardb-cluster.md).
+
+## Deploy ScalarDB Cluster
+
+```console
+helm install <release name> scalar-labs/scalardb-cluster -n <namespace> -f /path/to/<your custom values file for ScalarDB Cluster>
+```
+
+## Upgrade the deployment of ScalarDB Cluster
+
+```console
+helm upgrade <release name> scalar-labs/scalardb-cluster -n <namespace> -f /path/to/<your custom values file for ScalarDB Cluster>
+```
+
+## Delete the deployment of ScalarDB Cluster
+
+```console
+helm uninstall <release name> -n <namespace>
+```
+
+## Deploy your client application on Kubernetes with `direct-kubernetes` mode
+
+If you use ScalarDB Cluster with `direct-kubernetes` mode, you have to:
+
+* Deploy your application pods on the same Kubernetes cluster as ScalarDB Cluster.
+* Create three Kubernetes resources (`Role`, `RoleBinding`, and `ServiceAccount`).
+* Mount the `ServiceAccount` on your application pods.
+
+This is because the ScalarDB Cluster client library with `direct-kubernetes` mode runs Kubernetes API from inside of your application pods to get the information of ScalarDB Cluster pods.
```

Comment:
I add a requirement about how to deploy users' applications with `direct-kubernetes` mode here.
However, I think this is a description of ScalarDB Cluster (client library) itself.
In other words, this is not a Helm Chart specific thing.
So, it might be better to explain this in the ScalarDB Cluster's document.

@brfrn169  
What do you think?
(If we should explain this in the ScalarDB Cluster's document, I will copy this section to the `README` of ScalarDB Cluster. After that, I will add link to that document here.)

Reply:
> it might be better to explain this in the ScalarDB Cluster's document.

Agreed.

I'm going to reorganize the document soon. So I will add this explanation at that time. Thanks.

Reply:
> I'm going to reorganize the document soon. So I will add this explanation at that time. Thanks.

Thank you!

At this time, I remain these steps here, but I will update this document after you reorganize the ScalarDB Cluster's document.